frontend/public/components/filter-toolbar.tsx

```diff
@@ -224,7 +225,7 @@ const FilterToolbar_: React.FC<FilterToolbarProps & RouteComponentProps> = (prop
 
   const clearAll = () => {
     updateRowFilterSelected(selectedRowFilters);
-    if (!hideToolbar) {
+    if (!hideNameLabelFilters || !hideLabelFilter) {
       updateNameFilter('');
     }
     if (!hideLabelFilter) {
```

Comment:
```suggestion
    if (!hideNameLabelFilters || !hideLabelFilter) {
```
since the label filter can be hidden by both, no?